### ISSUE TITLE
Diff in GAP master branch

### ISSUE THREAD

**Issue by alex-konovalov** [gap-packages/numericalsgps#56]
See https://travis-ci.org/gap-infra/gap-docker-pkg-tests-master/jobs/645232348
```
########> Diff in /home/gap/inst/gap-master/pkg/NumericalSgps-1.2.1/tst/testal\
l.tst:1054
# Input is:
SmallElements(s);
# Expected output:
[ 0, 4, 8, 12, 16, 20 ]
# But found:
[ 0, 4 .. 20 ]
########
########> Diff in /home/gap/inst/gap-master/pkg/NumericalSgps-1.2.1/tst/testal\
l.tst:1328
# Input is:
SmallElements(I);
# Expected output:
[ 3, 5, 7, 9, 11, 13 ]
# But found:
[ 3, 5 .. 13 ]
########
```
Note that this still works in stable-4.11, so adjusting the test to new output will break the tests there. You can use `AsList` to make the test pass in both.

**Comment by pedritomelenas** [comment]
We do not see how to solve it with `AsList` ; we are using `SmallElements(s)=[ 0, 4, 8, 12, 16, 20 ];` instead.

**Comment by pedritomelenas** [comment]
Solved in 061fc46

**Comment by alex-konovalov** [comment]
Thanks - then I will expect that this will be fixed when I will pick up the new release.
